addons/udes_stock/models/stock_move_line.py

```diff
@@ -621,3 +623,72 @@ def _split_and_group_mls_by_quantity(self, maximum_qty):
             remaining_mls -= mls_used
             grouped_mls.append(mls_used)
         return grouped_mls
+
+    def _merge_move_lines(self):
+        """ This method will, for each move line in `self`, go up in their linked picking and try to
+        find in their existing move line a candidate into which we can merge the move.
+        :return: Recordset of move lines passed to this method.
+        If some of passed move lines were merged into another existing one, return this one and
+        not the (now unlinked) original.
```

Comment:
nit: Can we move this comment above `:return: Recordset of move lines passed to this method`?

Reply:
oh I see, this is part of the description of the `:return:`. I will resolve then :slightly_smiling_face: